packages/@react-aria/menu/src/useMenuTrigger.ts

```diff
@@ -56,12 +56,10 @@ export function useMenuTrigger(props: MenuTriggerAriaProps, state: MenuTriggerSt
         case 'Enter':
         case ' ':
           e.preventDefault();
-          e.stopPropagation();
```

Comment:
Perhaps it would be safer to do a check if propagation has already been stopped and skipping the stopPropagation call instead of completely removing it here.
Something like this maybe?
```
if (!e.isPropagationStopped()) {
  e.stopPropagation();
}
```

Reply:
Yep, my mistake! I should have checked for it. Thanks for the comment, I'll make the changes